test/e2e/specs/editor/editor__page-basic-flow.ts

```diff
@@ -60,12 +61,16 @@ describe( DataHelper.createSuiteTitle( 'Editor: Basic Post Flow' ), function ()
 		editorPage = new EditorPage( page );
 		// Allow some time for CPU and/or network to catch up.
 		await editorPage.selectTemplateCategory( 'About', { timeout: 20 * 1000 } );
-		await editorPage.selectTemplate( 'About Page 3', { timeout: 15 * 1000 } );
+
+		const editorParent = await editorPage.getEditorParent();
+		pageTemplateToSelect =
+			( await editorParent.getByRole( 'option' ).first().getAttribute( 'aria-label' ) ) ?? '';
```

Comment:
Added the reference for the latest update: https://github.com/Automattic/wp-calypso/commit/8ea0d9fb5025711f60c3cdbc16d967f5d2d05b0d

Reply:
Note that we ultimately went with a more semantic approach: 13fe701